### PR TITLE
Permit nested multi with block

### PR DESCRIPTION
**Pull request by JeffLuckett**
Redis 4.0+ supports nested multi in block context

Addresses: https://github.com/brigade/mock_redis/issues/150

### CODE REVIEW

File: lib/mock_redis/version.rb [hunk 1]
```diff
@@ -2,5 +2,5 @@
 
 # Defines the gem version.
 class MockRedis
-  VERSION = '0.19.0'.freeze
+  VERSION = '0.20.0'.freeze
```
Comment (sds) [inline lib/mock_redis/version.rb]: Please don't bump the gem version. We'll do that separately.

**Comment by JeffLuckett** [comment]
@sds - Thanks ... didn't want to put the effort into the rubocop cleanup until I was sure you were interested in the PR.

I'll get the build passing ... thanks!

File: lib/mock_redis/version.rb (hunk 1 above)
Comment (JeffLuckett) [reply lib/mock_redis/version.rb]: Got it.

**Comment by JeffLuckett** [comment]
@sds - I have cleaned up the rubocop violations that were flagged.  However, it seems Travis is having an issue connecting to Redis ... which is bombing the rspec tests.

I pass all specs locally against Ruby 2.5.1 and Redis 4.0.6 (as installed on my machine).

Can you advise on how to proceed if/when the Travis issues are resolved?

**Comment by JeffLuckett** [comment]
Ok @sds - sorry for the stream of messages 😄 .

I fixed your Redis issue on Travis: (See here for resolution used https://travis-ci.community/t/trusty-environment-redis-server-not-starting-with-redis-tools-installed/650)

My build is now passing.

**Pull request merged**